src/tzdata/TZData.jl

```diff
@@ -14,11 +14,17 @@ end
 # the "tzdata" archive or more specifically the "tz source" files within the archive
 # (africa, australasia, ...)
 
-const ARCHIVE_DIR = joinpath(DEPS_DIR, "tzarchive")
-const TZ_SOURCE_DIR = joinpath(DEPS_DIR, "tzsource")
-const COMPILED_DIR = joinpath(DEPS_DIR, "compiled", string(VERSION))
+function _init()
+    global ARCHIVE_DIR = joinpath(TimeZones.DEPS_DIR, "tzarchive")
+    global TZ_SOURCE_DIR = joinpath(TimeZones.DEPS_DIR, "tzsource")
+    global COMPILED_DIR = joinpath(TimeZones.DEPS_DIR, "compiled", string(VERSION))
+    global ACTIVE_VERSION_FILE = joinpath(TimeZones.DEPS_DIR, "active_version")
+    global LATEST_FILE = joinpath(TimeZones.DEPS_DIR, "latest")
```

Comment:
I think I'd prefer if these were all global constants and we used `Ref{String}()`

Reply:
Are any of these constants regarded as public? It'll of course be a breaking change if that's the case unfortunately.